rules/windows/defense_evasion_hide_encoded_executable_registry.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/11/25"
```

Comment:
@w0rk3r this rule is very expensive for performance (checking every registry.data.strings for every regmod event) and should be deprecated.

Reply:
Any room for logic adjusts here? Deprecating it wouldn't take effect into the the performance impacts for the existing customers